hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCommits.java

```diff
@@ -397,6 +397,30 @@ public void testLockExceptionUnknownSuccessCommit() throws TException, Interrupt
         .isTrue();
   }
 
+  @Test
+  public void testCommitExceptionWithoutMessage() throws TException, InterruptedException {
```

Comment:
can be simplified to
```
@Test
  public void testCommitExceptionWithoutMessage() throws TException, InterruptedException {
    Table table = catalog.loadTable(TABLE_IDENTIFIER);
    HiveTableOperations ops = (HiveTableOperations) ((HasTableOperations) table).operations();

    TableMetadata metadataV1 = ops.current();
    table.updateSchema().addColumn("n", Types.IntegerType.get()).commit();

    ops.refresh();

    HiveTableOperations spyOps = spy(ops);
    doThrow(new RuntimeException()).when(spyOps).persistTable(any(), anyBoolean(), any());

    assertThatThrownBy(() -> spyOps.commit(ops.current(), metadataV1))
        .isInstanceOf(CommitStateUnknownException.class)
        .hasMessageStartingWith("null\nCannot determine whether the commit was successful or not");
  }
```

Reply:
> @Test
>   public void testCommitExceptionWithoutMessage() throws TException, InterruptedException {
>     Table table = catalog.loadTable(TABLE_IDENTIFIER);
>     HiveTableOperations ops = (HiveTableOperations) ((HasTableOperations) table).operations();
> 
>     TableMetadata metadataV1 = ops.current();
>     table.updateSchema().addColumn("n", Types.IntegerType.get()).commit();
> 
>     ops.refresh();
> 
>     HiveTableOperations spyOps = spy(ops);
>     doThrow(new RuntimeException()).when(spyOps).persistTable(any(), anyBoolean(), any());
> 
>     assertThatThrownBy(() -> spyOps.commit(ops.current(), metadataV1))
>         .isInstanceOf(CommitStateUnknownException.class)
>         .hasMessageStartingWith("null\nCannot determine whether the commit was successful or not");
>   }

Thank you very much, I have simplified unit test.